core/src/test/java/software/amazon/awssdk/core/config/ImmutableClientConfigurationTest.java

```diff
@@ -45,17 +45,11 @@
     private static final SdkAsyncHttpClient ASYNC_HTTP_CLIENT = mock(SdkAsyncHttpClient.class);
     private static final ExecutionInterceptor EXECUTION_INTERCEPTOR = new ExecutionInterceptor() {
     };
-    private static final RetryPolicy RETRY_POLICY = new RetryPolicy() {
-        @Override
-        public long computeDelayBeforeNextRetry(RetryPolicyContext context) {
-            return 0;
-        }
 
-        @Override
-        public boolean shouldRetry(RetryPolicyContext context) {
-            return false;
-        }
-    };
+    private static final RetryPolicy RETRY_POLICY = RetryPolicy.builder()
+                                                               .retryCondition((c) -> false)
+                                                               .backoffStrategy((c) -> Duration.ZERO)
```

Comment:
Do we need the extra paranthesis around c? Also better naming for c?

Reply:
Nope and sure